deploy/addons/gvisor/gvisor-containerd-shim.toml

```diff
@@ -0,0 +1,5 @@
+runc_shim = "/bin/containerd-shim"
+[runsc_config]
```

Comment:
It might be useful to add the user-log option.

    user-log="/tmp/runsc/user-log-%ID%.log"